lib/internal/timers.js

```diff
@@ -1,15 +1,11 @@
 'use strict';
 
-const async_wrap = process.binding('async_wrap');
-// Two arrays that share state between C++ and JS.
-const { async_hook_fields, async_id_fields } = async_wrap;
 const {
   getDefaultTriggerAsyncId,
-  // The needed emit*() functions.
+  newUid,
+  initHooksExist,
   emitInit
 } = require('internal/async_hooks');
-// Grab the constants necessary for working with internal arrays.
-const { kInit, kAsyncIdCounter } = async_wrap.constants;
 // Symbols for storing async id state.
 const async_id_symbol = Symbol('asyncId');
```

Comment:
Why do we have separate symbols for this 😂. I think they should just be `async_id_symbol` and `trigger_async_id_symbol` from `async_hooks`.

Reply:
Funny enough, it's actually necessary because of enroll and unenroll. Since they could use an object that already has any asyncId associated, it would overwrite it or misrepresent it in emit.